docs/strategy-callbacks.md

```diff
@@ -1006,6 +1006,8 @@ Values that are above `max_leverage` will be adjusted to `max_leverage`.
 For markets / exchanges that don't support leverage, this method is ignored.
 
 ``` python
+from typing import Optional
```

Comment:
that seems rather picky if i'm honest
why should `Optional` be imported, but `datetime` (for this example) not? 


i think we'll need to decide one way or another ... (currently it's kinda a mix)
* either - we want all partial samples (like - all calback samples) to contain all imports
* or - we don't - in which case, we shouldn't have any "default" imports in the samples

Reply:
Agreed - this was to bring it in line with the other examples, and I think that partial imports are fine as they help people who are unfamiliar to understand what extra things are needed, and more experienced people will probably have them already.